### PR TITLE
Method to retrieve SIM's ICCID

### PR DESCRIPTION
**Pull request by ubidefeo**
- implemented `getICCID()` method to retrieve SIM ICCID number as a String

### CODE REVIEW

**Comment by ubidefeo** [comment]
@facchinm @sandeepmistry can this be merged in?
thanks :)

File: src/GSMModem.h [hunk 1]
```diff
@@ -37,6 +37,7 @@ class GSMModem {
       @return modem IMEI number
    */
   String getIMEI();
+  String getICCID();
```
Comment (sandeepmistry) [inline src/GSMModem.h]: If you can add a comment like the `getIMEI()` one above, that would be great.

Comment (ubidefeo) [reply src/GSMModem.h]: done and done :)
thank you, @sandeepmistry

**Comment by sandeepmistry** [comment]
@ubidefeo would you also like to port this to [MKRNB](https://github.com/arduino-libraries/MKRNB)?

**Comment by ubidefeo** [comment]
@sandeepmistry sure, it's a two minutes thing if the command is supported by the modem :)

File: src/GSMModem.cpp [hunk 2]
```diff
@@ -45,3 +45,15 @@ String GSMModem::getIMEI()
...
 }
+
+String GSMModem::getICCID()
+{
+  String iccid;
+
+  iccid.reserve(20);
+
+  MODEM.send("AT+CCID");
+  MODEM.waitForResponse(120, &iccid);
```
Comment (sandeepmistry) [inline src/GSMModem.cpp]: @ubidefeo I think we need to bump this timeout from 120, to something higher - maybe 1000?

**Pull request merged**